rust/phoenix-sdk/src/order_packet_template.rs

```diff
@@ -33,3 +34,91 @@ pub struct LimitOrderTemplate {
     /// If this is set, the order will be invalid after the specified unix timestamp
     pub last_valid_unix_timestamp_in_seconds: Option<u64>,
 }
+
+/// PostOnlyOrderTemplate is a helper type for creating a post-only order, which will never be matched against existing orders.
+/// The template allows you to specify the price and size in commonly understood units:
+/// (i) price is simply the floating point price as represented on most exchanges, and (ii) size is the number of base whole units (i.e. one SOL token).
+/// The SDK can then convert this to a post-only order instruction, ready to be sent.
+pub struct PostOnlyOrderTemplate {
+    // The side for the order, a Side::Bid or a Side::Ask
+    pub side: Side,
+
+    /// The price of the order, as the commonly understood exchange price (the number of quote units to exchange for one base unit), as a floating point number
+    pub price_in_float: f64,
```

Comment:
`price_as_float`? Maybe 'float_price`?

Reply:
Agreed. In the TS SDK, Eugene had the good suggestion to use the camel case equivalent of `price_as_float` instead of `price_in_float` and `size_in_base_units` instead of `num_base_units`. I changed the fields here to match that pattern too.